packages/create-modular-react-app/src/cli.ts

```diff
@@ -5,16 +5,23 @@ import { program } from 'commander';
 import * as fs from 'fs-extra';
 import { JSONSchemaForNPMPackageJsonFiles as PackageJson } from '@schemastore/package';
 
+// this is a bit gross - but there's no better way of doing this.
+let verbose = false;
+
 // Makes the script crash on unhandled rejections instead of silently
 // ignoring them. In the future, promise rejections that are not handled will
 // terminate the Node.js process with a non-zero exit code.
-// See https://github.com/facebook/create-react-app/blob/f36d61a/packages/react-scripts/bin/react-scripts.js#L11-L16
 process.on('unhandledRejection', (err) => {
-  throw err;
+  if (verbose) {
```

Comment:
Given that you catch both async func calls, I don't think this event is likely to happen. But I also think we should just throw the entire err regardless of verbose or not if we do encounter an unhandledRejection error. Let it break so we can fix it.

Reply:
yeah - I can live with this.